src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -387,13 +387,7 @@ private function getArgumentsAsPhp(\DOMElement $node, $name, $lowercase = true)
                         $invalidBehavior = ContainerInterface::NULL_ON_INVALID_REFERENCE;
                     }
 
-                    if ($strict = $arg->getAttribute('strict')) {
```

Comment:
But there is no way to create any continuous upgrade path in XML files. That's why we didn't remove "strict" from the XSD, to allow cross 2x3 bundles. Should we consider this is not required anymore? Not sure on my side (because it doesn't hurt). WDYT?